src/components/Request/Pay/Views/Initial.view.tsx

```diff
@@ -9,13 +9,19 @@ import Icon from '@/components/Global/Icon'
 import MoreInfo from '@/components/Global/MoreInfo'
 import * as consts from '@/constants'
 import { useCreateLink } from '@/components/Create/useCreateLink'
-import { peanut } from '@squirrel-labs/peanut-sdk'
+import { peanut, interfaces } from '@squirrel-labs/peanut-sdk'
 import TokenSelector from '@/components/Global/TokenSelector/TokenSelector'
 import { switchNetwork as switchNetworkUtil } from '@/utils/general.utils'
-import { ADDRESS_ZERO, EPeanutLinkType, RequestStatus } from '../utils'
 
 const ERR_NO_ROUTE = 'No route found to pay in this chain and token'
 
+enum RequestStatus {
+    LOADING = 'LOADING',
+    CLAIM = 'CLAIM',
+    NOT_CONNECTED = 'NOT_CONNECTED',
+    NOT_FOUND = 'NOT_FOUND',
+}
```

Comment:
Question: Is there an explainer somewhere with these statuses? How is claim a status?

Reply:
no explainer, and is only used in this view. I agree could use better names